rclcpp/include/rclcpp/qos.hpp

```diff
@@ -145,11 +145,39 @@ class RCLCPP_PUBLIC QoS
   // cppcheck-suppress noExplicitConstructor
   QoS(size_t history_depth);  // NOLINT(runtime/explicit): conversion constructor
 
-  /// Return the rmw qos profile.
```

Comment:
Nitpick-y note: please keep this line. We usually have 3 slashes followed by a one-line summary of the function, and then we put more details below like you did. See the docs for the other functions

Reply:
Done!